pkg/generator/generator.go

```diff
@@ -51,9 +52,16 @@ func New(log logrus.FieldLogger, s3Client s3wrapper.API, cfg Config, workDir str
 // GenerateInstallConfig creates install config and ignition files
 func (k *installGenerator) GenerateInstallConfig(ctx context.Context, cluster common.Cluster, cfg []byte, releaseImage string) error {
 	log := logutil.FromContext(ctx, k.log)
-	clusterWorkDir := filepath.Join(k.workDir, cluster.ID.String())
+	err := os.MkdirAll(k.workDir, 0o755)
```

Comment:
What's the meaning of the `o` in `0o755`?

Reply:
octal